src/plots/cartesian/set_convert.js

```diff
@@ -360,7 +361,6 @@ module.exports = function setConvert(ax) {
             Lib.notifier(
                 'Something went wrong with axis scaling',
                 'long');
-            ax._gd._replotting = false;
             throw new Error('axis scaling');
```

Comment:
Sounds good. I'll try to cook up a test case for this edge case. Any help would be appreciated ;)